docs/setup/installation.md

```diff
@@ -496,6 +496,16 @@ Cross-Origin Resource Sharing (CORS) headers. A recommended value would be
 `Access-Control-Allow-Origin: *` which would allow all browser based clients to
 view it.
 
+Synapse will serve it once you set `public_baseurl` in your `homeserver.yaml`.
+
+Just like:
+
+```yaml
+public_baseurl: "https://<matrix.example.com>"
+```
+
+Alternatively, you can set your reverse proxy to serve it.
```

Comment:
```suggestion
Synapse will generate a response if `public_baseurl` is configured in `homeserver.yaml`:

```yaml
public_baseurl: "https://<matrix.example.com>"
```

Alternatively, you can set your reverse proxy to serve it.
```

Reply:
Should this cross-reference the configuration manual w/ the various settings it can include?